Test/Optimizer/test_sr.py

```diff
@@ -18,36 +18,25 @@ def test_svd_threshold():
     """
     Test SVD threshold option of BDCSVD
     """
-    with pytest.raises(
-        ValueError, match="svd_threshold option only available for BDCSVD solver"
-    ):
-        SR(svd_threshold=1e-3)
-    with pytest.raises(
-        ValueError, match="svd_threshold option only available for BDCSVD solver"
-    ):
+    with pytest.raises(ValueError):
```

Comment:
Could you match on the correct error message? Many things can throw a `ValueError`, we should check the error is the correct one.